mslib/msui/autoplot_dockwidget.py

```diff
@@ -27,15 +27,17 @@
     limitations under the License.
 """
 
-import json
 import os
-import click
-from mslib.utils.mssautoplot import main as cli_tool
+import json
+from datetime import datetime
+
 from PyQt5.QtWidgets import QWidget, QFileDialog, QTreeWidgetItem, QMessageBox
 from PyQt5 import QtCore
+import click
```

Comment:
move this before the first from PyQt5 import then we have all from after each other. both are also not in the standardlib